tools/vm_sanitize_enforcement.py

```
@@ -112,7 +112,8 @@ def main():
 			return ERROR
 		header = filename[:-1] + "h"
 
-		are_safe_types_used = are_safe_types_used_in_file(header)
+		if are_safe_types_used_in_file(header):
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)